terraform/cluster.tf

```diff
@@ -5,17 +5,12 @@ resource "azurerm_kubernetes_cluster" "bakery_cluster" {
   dns_prefix          = "${var.identifier}-bakery-cluster"
 
   default_node_pool {
-    name                = "default"
-    max_count           = 100
-    min_count           = 1
-    vm_size             = "standard_d4_v4"
-    os_disk_size_gb     = 512
-    enable_auto_scaling = true
-  }
-
-  auto_scaler_profile {
-    scale_down_delay_after_add = "2m"
-    scale_down_unneeded        = "2m"
+    name            = "system"
```

Comment:
Just a note that changing this *might* delete and recreate the entire kubernetes cluster. Hopefully that's OK.

Reply:
It will indeed, but the code in this repo is idempotent, so this shouldn't affect much.

Reply:
(IE the manifests etc, so rerunning should only yield a re-spin)